docs/Getting-Started-Guides/On-Premises-FPGA-Getting-Started/Initial-Setup/Xilinx-XDMA-Template.rst

```diff
@@ -49,27 +49,115 @@ To do so, edit your ``~/.bashrc`` file so that the following section is removed:
           *) return;;
    esac
 
+2. Install/enable FireSim scripts to new ``firesim`` Linux group
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-2. Enable password-less sudo
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**Machines:** Manager Machine, Run Farm Machines, Build Farm Machines.
+
+First, let's clone a temporary version of FireSim with the scripts within it:
+
+.. code-block:: bash
+   :substitutions:
+
+   cd ~/     # or any scratch directory
+   mkdir firesim-script-installs
+   cd firesim-script-installs
+   git clone https://github.com/firesim/firesim
+   cd firesim
+   # checkout latest official firesim release
+   # note: this may not be the latest release if the documentation version != "stable"
+   git checkout |overall_version|
+
+Next, copy the required scripts to  ``/usr/local/bin``:
+
+.. code-block:: bash
+   :substitutions:
+
+   sudo cp deploy/sudo-scripts/* /usr/local/bin
+   sudo cp platforms/xilinx_alveo_u250/scripts/* /usr/local/bin
+
+Next, lets change the permissions of the scripts and them to a new ``firesim`` Linux group.
+
+.. code-block:: bash
+   :substitutions:
+
+   sudo addgroup firesim
+   sudo chmod 775 /usr/local/bin/firesim*
+   sudo chgrp firesim /usr/local/bin/firesim*
+
+Next, lets allow the ``firesim`` Linux group to run the pre-installed commands.
+Enter/create the following file with `sudo`:
+
+.. code-block:: bash
+   :substitutions:
+
+   sudo visudo /etc/sudoers.d/firesim
+
+Then add the following lines:
+
+.. code-block:: bash
+   :substitutions:
+
+   %firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-change-pcie-perms
```

Comment:
Try to glob instead of specifying file names exactly (esp. since all start with `firesim-*`)

Reply:
Testing: `%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-*`

Reply:
Worked. Will update.